packages/react-components/src/components/ControlBarButton.tsx

```diff
@@ -133,19 +133,19 @@ export const ControlBarButton = (props: ControlBarButtonProps): JSX.Element => {
       ? props?.strings?.tooltipOnContent
       : props?.strings?.tooltipOffContent);
 
+  const splitButtonAriaLabel = props.splitButtonAriaLabel ? tooltipContent + props.splitButtonAriaLabel : undefined;
+
   const tooltipId = props.tooltipId ?? props.labelKey ? props.labelKey + '-tooltip' : undefined;
 
   return (
     <ControlButtonTooltip content={tooltipContent} id={tooltipId}>
       <DefaultButton
         {...props}
-        role={props.split ? 'menu' : undefined}
         styles={componentStyles}
         onRenderText={props.showLabel && props.onRenderText ? props.onRenderText : undefined}
         onRenderIcon={props.onRenderIcon ?? DefaultRenderIcon}
-        ariaLabel={props.ariaLabel ?? tooltipContent ?? labelText}
+        ariaLabel={!splitButtonAriaLabel ? props.ariaLabel ?? tooltipContent ?? labelText : splitButtonAriaLabel}
         allowDisabledFocus={props.allowDisabledFocus ?? true}
-        aria-roledescription={props.ariaDescription}
```

Comment:
Revert this? Or is this intentionally left deleted?

Reply:
Actually I agree to removing this. With this when `aria-roledescription` is passed into ControlBarButton it is applied twice; once as role-description and once as ariaDescription. So sounds OK to remove it.. but what about other buttons that are using ariaDescription?

Reply:
this was intentionally deleted since we pass it in from the camera and microphone button. It didn't make sense to have this work this way since the control bar button and the camera and mic buttons both have `aria-roledescription` and `ariaDescription` props and we cross where that information is coming from here.

Reply:
Ah no other buttons use this yet! Sounds good 👍

Reply:
We do have some other places we may need to update if aria-roledesciption is preferred over ariaDescription
(but _not_ in this PR!)
![image](https://user-images.githubusercontent.com/2684369/165570790-2783b6b3-1ec8-4d74-95c5-e8743d36c8ab.png)